cypress/e2e/pages/articles/testsForLiteOnly.js

```diff
@@ -0,0 +1,10 @@
+/* eslint-disable import/prefer-default-export */
+export const testsForLiteOnly = ({ service, pageType }) => {
+  describe(`Running testsForLiteOnly for ${service} ${pageType}`, () => {
+    describe('CTA: Lite', () => {
+      it('should render a call to action component', () => {
+        cy.get('[data-e2e="lite-cta"]').should('be.visible');
```

Comment:
I know that this is now merged, but I do wonder if this particular assertion might have been better/faster as an integration test (are there plans to add integration tests for a lite page?) 

In my opinion, the E2E test for the lite page should actually click on the lite CTA & assert that we ended up on the right page? @paruchurisilpa and @amoore108 do you have any thoughts on this?